setup.py

```diff
@@ -11,6 +11,10 @@
 with open('requirements.txt') as f:
     requirements = f.read().splitlines()
 
+requirements_dev = []
+with open('requirements.txt') as f:
```

Comment:
And this was meant to read the file `requirements_dev.txt` correct?

Reply:
same as above - just fixed it in the latest commit